spec/services/integration_tests/test_runner_spec.rb

```diff
@@ -18,12 +18,35 @@ def post_resource(url, params)
       expect(JSON.parse(response.body)['success']).to eq true
     end
 
+    def fetch_result(assessment_id)
+      get assessment_path(assessment_id), headers: headers
+      JSON.parse(response.body, symbolize_names: true)
+    end
+
+    def compare_result(assessment_id, payload, spreadsheet_name) # rubocop:disable Lint/UnusedMethodArgument
+      result = fetch_result(assessment_id) # rubocop:disable Lint/UselessAssignment
+
+      # TODO: uncomment to see difference between actual and expected results
+      # p '---------'
```

Comment:
I think you should be using `puts` rather than `p` for most of these lines. For example, that would make the output from this line:
```
---------
```
Rather than the current:
```
"---------"
```